test/e2e/gameserver_test.go

```diff
@@ -0,0 +1,69 @@
+// Copyright 2018 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package e2e
+
+import (
+	"fmt"
+	"testing"
+
+	"agones.dev/agones/pkg/apis/stable/v1alpha1"
+	agonesFramework "agones.dev/agones/test/e2e/framework"
```

Comment:
Any reason we can't use "framework"?

Pretty sure it's not canonical go to use camelCase here. @enocom I'll defer to you?

Looking at https://golang.org/doc/effective_go.html#package-names

"By convention, packages are given lower case, single-word names; there should be no need for underscores or mixedCaps"

"Another convention is that the package name is the base name of its source directory; the package in src/encoding/base64 is imported as "encoding/base64" but has name base64, not encoding_base64 and not encodingBase64."

But I couldn't find anything specific about import aliases.

Reply:
Yep, @markmandel, you're right. Named imports should be all lowercase.

Reply:
`framework` is used by a global variable, do you think I should not use a global variable ? I could renamed that import to `e2eframework` ?